### PR TITLE
Add color palette for data visualization

### PR DESCRIPTION
**Pull request by krkshitij**
- Update styles as per design
- Add high contrast and RTL styles
- Add DataVizPalette
- Implement color fallback from DataVizPalette for arcs
- Assign appropriate role and aria attributes to legends

### CODE REVIEW

File: packages/web-components/src/donut-chart/donut-chart.options.ts [hunk 1]
```diff
@@ -36,5 +36,5 @@ export interface IChartProps {
   /**
    * data for the points in the chart
    */
-  chartData?: IChartDataPoint[];
+  chartData?: ChartDataPoint[];
```
Comment (AtishayMsft) [inline packages/web-components/src/donut-chart/donut-chart.options.ts:39]: make this change for horizontal bar chart as well

File: packages/web-components/src/utils/chart-helpers.ts [hunk 2]
```diff
@@ -60,3 +60,147 @@ const validateChartProps = (obj: any) => {
...
+  '4': ['#9373c0'], // [orchid.tint10],
+  '5': ['#13a10e'], // [lightGreen.primary],
+  '6': ['#3a96dd'], // [lightBlue.primary],
+  '7': ['#ca5010'], // [pumpkin.primary],
+  '8': ['#57811b'], // [lime.shade20],
+  '9': ['#b146c2'], // [lilac.primary],
+  '10': ['#ae8c00'], // [gold.shade10],
+  '11': ['#3c51b4', '#93a4f4'], // [cornflower.shade20, cornflower.tint30],
+  '12': ['#ad006a', '#ee5fb7'], // [hotPink.shade20, hotPink.tint30],
+  '13': ['#026467', '#4cb4b7'], // [teal.shade20, teal.tint30],
```
Comment (AtishayMsft) [inline packages/web-components/src/utils/chart-helpers.ts:135]: remove 11-20 and 30-40. We will add them back once we have a solution for dark mode.

File: packages/web-components/src/donut-chart/donut-chart.ts [hunk 3]
```diff
@@ -39,6 +42,17 @@ export class DonutChart extends FASTElement {
 
   connectedCallback() {
     super.connectedCallback();
+
+    this._isRTL = getDirection(this) === Direction.rtl;
```
Comment (AtishayMsft) [inline packages/web-components/src/donut-chart/donut-chart.ts:46]: make this a utility

File: packages/web-components/src/donut-chart/donut-chart.options.ts (hunk 1 above)
Comment (krkshitij) [reply packages/web-components/src/donut-chart/donut-chart.options.ts:39]: done

File: packages/web-components/src/donut-chart/donut-chart.ts (hunk 3 above)
Comment (krkshitij) [reply packages/web-components/src/donut-chart/donut-chart.ts]: done

File: packages/web-components/src/utils/chart-helpers.ts (hunk 2 above)
Comment (krkshitij) [reply packages/web-components/src/utils/chart-helpers.ts]: done

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**